plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/AbstractTrinoCatalog.java

```diff
@@ -288,6 +291,13 @@ protected ConnectorMaterializedViewDefinition getMaterializedViewDefinition(
             properties.put(PARTITIONING_PROPERTY, toPartitionFields(icebergTable.spec()));
         }
 
+        if (!icebergTable.location().isEmpty()) {
```

Comment:
can `icebergTable.location()` be an empty string? when?

Reply:
> This `location` is actually the location of the storage table, when the storage table is dropped and a storage table with an empty location with the same name is created, the location will be empty.

This is not a valid situation we need to care about.
If a storage table is manually replaced with something else, all sorts of problems may occur.
`SHOW CREATE MATERIALIZED VIEW` may fail in such case.

Reply:
thus, please remove the condition

Reply:
done, thanks! 

Does this need to be modified? if so, I will submit another pr to modify it.
https://github.com/trinodb/trino/blob/master/plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java#L1296